obs_scripts/rsky.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import time
+from _observation import Observation
+
+
+class Rsky(Observation):
```

Comment:
Have you intentionally chosen this naming convention?
If not, please change the name to `RSky`, as there's no reason `s` to be lower case in CamelCase convention.

Reply:
No reason to choose the naming convention. I'll change according to your suggestion